### PR TITLE
Add diff ui

### PR DESCRIPTION
**Pull request by ga-devfront**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Add UI to see diff in SQL for autoupgrade-scripts
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Sponsor company   | @PrestaShopCorp
| How to test?      | Launch autoupgrade script and see the UI at the end of the script

### CODE REVIEW

File: autoupgrade-scripts/docker-compose.yml [hunk 1]
```diff
@@ -31,3 +31,17 @@ services:
         image: composer:latest
         volumes:
             - ./:/var/www/html/
+
+    visualize-diff-front:
+        build:
+            dockerfile: ../../SeamlessUpgradeToolbox/autoupgrade-scripts/visualize-diff-front.DockerFile
```
Comment (M0rgan01) [inline autoupgrade-scripts/docker-compose.yml:37]:
```suggestion
            dockerfile: visualize-diff-front.DockerFile
```

File: autoupgrade-scripts/docker-compose.yml [hunk 2]
```diff
@@ -31,3 +31,17 @@ services:
...
+        ports:
+            - "8080:80"
+
+    visualize-diff-back:
+        build:
+            dockerfile: ../../SeamlessUpgradeToolbox/autoupgrade-scripts/visualize-diff-back.DockerFile
```
Comment (M0rgan01) [inline autoupgrade-scripts/docker-compose.yml:43]: dockerfile: visualize-diff-back.DockerFile

**Comment by Quetzacoalt91** [comment]
Do we need to keep this new app with the docker compose files previously added by @M0rgan01? Or can they go in a new folder?

File: autoupgrade-scripts/upgrade.sh [hunk 3]
```diff
@@ -284,3 +284,18 @@ fi
 
 docker compose up -d prestashop-run
 echo "--- Docker container created for upgrade, see result at http://localhost:8002/admin ---"
+
+FRONT_CONTAINER_NAME="visualize-diff-front"
+BACK_CONTAINER_NAME="visualize-diff-back"
```
Comment (M0rgan01) [inline autoupgrade-scripts/upgrade.sh:289]: I think we can initialize them at the top, with the other variables

File: autoupgrade-scripts/visualize-diff-front.DockerFile [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+# Use the lightweight Nginx image from the previous stage for the nginx container
+FROM nginx:stable-alpine as production-stage
+# Copy the build application from the previous stage to the Nginx container
+COPY --from=build-stage ./dist /usr/share/nginx/html
+# Copy the nginx configuration file
+COPY ./nginx/default.conf /etc/nginx/conf.d/default.conf
+# Expose the port 80
+EXPOSE 80
+# Start Nginx to serve the applicationa
+#CMD ["nginx", "-g", "daemon off;"]
```
Comment (M0rgan01) [inline autoupgrade-scripts/visualize-diff-front.DockerFile:18]: To remove or uncomment

**Pull request closed without merging**